v1/tarball/image.go

```diff
@@ -94,7 +105,33 @@ type singleManifest struct {
 // tarManifest is the struct used to represent a single image inside a `docker save` tarball.
 type tarManifest []singleManifest
 
+func findSpecifiedManifest(tag *name.Tag, tarMfst tarManifest) (*singleManifest, error) {
```

Comment:
Seems like it would read best with `tarMfst` (which I keeping reading as "Tar MSFT", so we should rename :D) as the receiver:
```golang
func (tm tarManifest) manifestForTag(tag *name.Tag) ...
```

WDYT?